src/stores/business.ts

```diff
@@ -36,6 +39,21 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
   const authApiURL = useRuntimeConfig().public.authApiURL
   const launchdarklyStore = useBcrosLaunchdarkly()
 
+  async function getBusinessComments(identifier: string) {
+    commentsLoading.value = true
+    comments.value = []
+    return await useBcrosFetch<CommentIF>(`${apiURL}/businesses/${identifier}/comments`, {})
+      .then(({ data, error }) => {
+        if (error.value || !data.value) {
+          console.warn('Error fetching comments for', identifier)
+          return null
+        }
+        comments.value = data.value.comments.map(comment => comment.comment)
+        commentsLoading.value = false
+        return comments.value
```

Comment:
I would either avoid returning anything, or use constant, and assign it on function return. This is creating confusing logic, where function has return value, but it returns value which is assigned to global ref.

Reply:
e.g.
```
 async function getBusinessComments(identifier: string) {
    commentsLoading.value = true
    return await useBcrosFetch<CommentIF>(`${apiURL}/businesses/${identifier}/comments`, {})
      .then(({ data, error }) => {
        if (error.value || !data.value) {
          console.warn('Error fetching comments for', identifier)
          return null
        }
        const comments = data.value.comments.map(comment => comment.comment)
        commentsLoading.value = false
        return comments
   })
  }
```
or just remove return

Reply:
I'll have to bother you to rename the function then :D to maybe fetchBusinessComments as get usually implies return value, sorry I didnt mentioned that previously